.github/workflows/build_cmake.yml

```diff
@@ -47,6 +45,15 @@ jobs:
             platform: mac_x64,
             cc: "clang", cxx: "clang++"
           }
+        qt_creator:
+        - {
+            version: "15.0.1",
```

Comment:
I am not sure about support old version, maybe are leave only current for now and in future add new like beta?

Reply:
Why not support older version as well? In our codebase the difference is 10 lines, it's not likely to raise any issues.

It's problematic to force users to upgrade to newest versions of Qt Creator because other third-party extensions may not upgrade as quickly. Also, new Qt Creator versions may contain bugs that break workflows and thus aren't usable. Both of these are barriers to use QodeAssist.

I would say that it makes sense to support all releases, especially since this PR has already done the work. If we actually see maintenance overhead, such as bugs that are specific to single version, then it makes sense to re-evaluate supporting fewer versions.

Reply:
In that case, we probably need to take the Qt version as part of it.

Reply:
I checked, yes, idea is good but it is lead more changes, for artifacts name, for check and download new version of plugin and plugins.json(indeed I don't know reason of this file like a collect data of plugins) and for usual PR we don't need to check all versions of QtC, only current I think. So lets do it later by another PR and just update now to new QtC version, because we don't have new features for now

Reply:
> for usual PR we don't need to check all versions of QtC, only current I think.

I think this doesn't matter because each build uses very little resources. It finishes in only couple of minutes.

Reply:
Agreed with the rest.

Reply:
> for check and download new version of plugin

This one already works.

Reply:
Not exactly, if we add new multiply versions, which version will be download, plus Qt version, QtCreator 15 was built on 6.8.1